.gitignore

```diff
@@ -29,3 +30,25 @@ dist-gem/
 # Ignore gulp-generated vendor files
 #
 src/stylesheets/lib
+docs/assets/fonts/
+docs/assets/js/vendor/uswds-0.9.0.js
+docs/_scss/
+docs/assets/img/alerts/
+docs/assets/img/arrow-down.png
```

Comment:
I would say though that we could _fix it and close an issue_ in the next release :wink: